quickstart updates

MBishop17 opened the pull request:
doc updates for beta quickstart guide

will-chan commented on rancher/v2.0/en/quick-start-guide/index.md:
```diff
@@ -11,86 +11,185 @@ redirect_from:
+
+   1. Enter a **Node Prefix**. When the cluster is created, each node in the pool is named after the prefix. An incremented number is appended to each node.
+   2. Enter the node **Count** for the pool.
+   3. Click **Add Node Template**. A node template is just the a virtual machine configuration you're using to create a docker image.
```
A node template is used to ultimately create a virtual machine/compute node, not a docker image.

will-chan commented on rancher/v2.0/en/quick-start-guide/index.md:
```diff
@@ -105,21 +204,56 @@ Rancher supports grouping resources into multiples clusters, projects and namesp
+
+A *project* is an object that fences in namespaces and workloads. We'll describe projects, namespaces, and workloads in more detail later.
+
+Out of the box, Rancher is bundled with a catalog of applications that make their deployment easy. Choose an application from the catalog for deployment.
```
we can say that the catalog of applications in the form of HELM charts...

MBishop17 commented:
@will-chan and @deniseschannon, I made completed edits per your feedback.

Denise, just a heads up--generally, I agree with your comments about skipping screen caps in documentation. However, since a Quick Start Guide is generally someone's first experience the software, I break that rule; I want to make it hand-holdy as possible. I'll take on that extra maintainence :) 

Have a good weekend guys!

deniseschannon commented on rancher/v2.0/en/quick-start-guide/index.md:
```diff
@@ -4,91 +4,248 @@ layout: rancher-default-v2.0
-  - /rancher/quick-start-guide/
-  - /rancher/v2.0/en/
+ /rancher/quick-start-guide/
+ /rancher/v2.0/en/
```
when you run this locally, do the redirects still work?

MBishop17 replied:
They do. Here's what happened: I installed a package in atom that beautifies markdown on save. Unfortunately, this package "beautified" the header as well--it added a bunch of garbage characters to the redirects so they weren't working. The update you see here is what I had after removing all the garbage characters. I tested them and they work.

deniseschannon commented on rancher/v2.0/en/quick-start-guide/index.md:
```diff
@@ -4,91 +4,248 @@ layout: rancher-default-v2.0
+
+	Use these options to choose things like the version of Kubernetes that's installed in your cluster, along with other Kubernetes options such as pod security policies. Some services have more options than others. If you're unsure of what to choose, use the default options.
+
+	**Using Google Container Engine?** Skip to the next step. GCE doesn't have any cluster options.
```
Google Container Engine's acronym is GKE. Also, GKE has cluster options after you enter your service account info. 

<img width="1142" alt="screen shot 2018-04-01 at 9 09 20 pm" src="https://user-images.githubusercontent.com/11410997/38182859-0f82bbb0-35f1-11e8-9868-e3bc3d84d292.png">

deniseschannon commented on rancher/v2.0/en/quick-start-guide/index.md:
```diff
@@ -4,91 +4,248 @@ layout: rancher-default-v2.0
+
+	A *Node Pool* is a group of nodes that are configured identically. Your cluster can contain as many node pools as you'd like. Each object in the grid represents a single node configuration. You can use the node pool to choose the number (i.e. **Count**) of nodes running a given configuration (i.e. **Template**).
+
+	> **Exception:** The instructions below don't apply to Azure Container Service. Node creation for Azure is a little different. Instead of accessing a template through an API, you must complete the **Nodes** form, which customizes nodes directly from Rancher.
```
We should add GKE for this exception as well.

MBishop17 commented:
@deniseschannon  and @will-chan, the quick start guide is ready for another review. I added an appendix for the custom option, and I updated the GKE instructions per your feedback Denise.
